build/images/codegen/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04 as protoc
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates unzip
+
+RUN PROTOBUF_VERSION=3.0.2; ZIPNAME="protoc-${PROTOBUF_VERSION}-linux-x86_64.zip"; \
+  mkdir /tmp/protoc && cd /tmp/protoc \
+  && wget "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/${ZIPNAME}" \
+  && unzip "${ZIPNAME}"
+
+FROM golang:1.13
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A Docker image based on golang 1.13 which includes codegen tools."
+
+ENV GO111MODULE=on
+
+RUN go get k8s.io/code-generator/cmd/client-gen@kubernetes-1.15.0 && \
```

Comment:
nit: can we define an `ARG` variable for the code generator version (`1.15.0`) so we can update it easily in one place if we need to in the future?

Reply:
Done